src/forms/settings-dialog.cpp

```diff
@@ -935,9 +939,10 @@ void PluginWindow::edit_mapping()
 		ui->cb_obs_output_audio_source->setCurrentText(selected_items.at(9)->text());
 		ui->cb_obs_output_media_source->setCurrentText(selected_items.at(10)->text());
 		const bool check = (selected_items.at(11)->text().toInt() > 0) ? true : false;
-		ui->cb_obs_output_hotkey->setCurrentText(selected_items.at(14)->text());
 		ui->check_int_override->setChecked(check);
 		ui->sb_int_override->setValue(selected_items.at(11)->text().toInt());
+		HotkeyModel *hotkeyModel = (HotkeyModel *)ui->cb_obs_output_hotkey->model();
+		ui->cb_obs_output_hotkey->setCurrentIndex(hotkeyModel->getIndexOfHotkeyDescription(selected_items.at(14)->text()));
```

Comment:
with the hotkey item, we dont need to getindexofhotkey etc. we can just setcurrent text to the hotkey items description.

Reply:
Don't think it works, gonna try it

Reply:
indeed, it worked 👌🏼